stdlib/public/core/UnsafeRawPointer.swift

```diff
@@ -1316,7 +1316,9 @@ public struct UnsafeMutableRawPointer: _Pointer {
   public func storeBytes<T : _BitwiseCopyable>(
     of value: T, toByteOffset offset: Int = 0, as type: T.Type
   ) {
+#if $BuiltinStoreRaw
     Builtin.storeRaw(value, (self + offset)._rawValue)
+#endif
```

Comment:
Might want an `#else` branch with `fatalError()` here to minimize surprises for users of older compilers